scripts/helpers/todos.js

```diff
@@ -0,0 +1,108 @@
+const path = require("path");
+const prettier = require("prettier");
+const TypeScript = require("typescript");
+
+const { readFile, writeFile } = require("./file-system");
+const { getLineAtOffset, parseLines } = require("./text");
+
+const { Project } = require("../helpers/project");
+
+/**
+ * @typedef {import("./text").Line} Line
+ */
+
+/**
+ * @typedef {Object} Todo
+ * @property {string} file
+ * @property {TypeScript.TodoComment} comment
+ */
+
+/**
+ * @type {Array<TypeScript.TodoCommentDescriptor>}
+ */
+const checks = [
+  { text: "@todo", priority: 1 },
+  { text: "@hack", priority: 1 },
+  { text: "@bug", priority: 1 },
+  { text: "@fixme", priority: 1 }
+];
+
+/**
+ * @param {string} file
+ * @param {Project} project
+ * @return {Iterable<Todo>}
+ */
+function getTodos(file, project) {
+  return [...project.getTodos(file, checks)].map(comment => ({
+    file,
+    comment
+  }));
+}
+
+exports.getTodos = getTodos;
```

Comment:
I think you can just write `export function getTodos(file, project) { ... `

Reply:
The module support in Node.js is both experimental and requires use of the `.mjs` file extension so all files in `scripts/` must use CJS.